examples/camera/app.js

```diff
@@ -55,6 +87,13 @@ export default function App() {
           // blendEquation: GL.FUNC_ADD,
           // blendFunc: [GL.SRC_ALPHA, GL.ONE_MINUS_SRC_ALPHA]
         }}
+
+        viewState={viewState}
+        onViewStateChange={({viewState}) => {
```

Comment:
Do you know if this addition and the useState hook for viewState is necessary still?

Is it needed because something needs to maintain the user's changes viewState? If so, could you add a comment to clarify its to maintain the user's interactions with viewState?

Reply:
I did duplicate the camera example to have something to work to start. 
About making 2 PRs: The **gltf-object-mapbox** example was my example from when we started. Should I make another branch without that example by pulling what's on master now, update my current changes and do just one PR?
I want to use the hubble-example because it has a mapbox and deckgl canvas, so we can try to implement the Mapbox Layer later.